definitions/divorce/json/CaseField/CaseField-nonprod.json

```diff
@@ -64,5 +64,61 @@
     "HintText": "The solicitor believes that the facts stated in this response are true",
     "FieldType": "YesOrNo",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "11/10/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelAmendPetitionForRefusalSol",
+    "Label": "## Submit amended application",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelAmendPetitionForRefusalSolPara-1",
+    "Label": "If you’ve been asked to submit an amended application by the courts, this must be completed for the divorce to proceed. Until this application has been completed no further action can be taken.\r\n\r\n",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelAmendPetitionForRefusalSolPara-2",
+    "Label": "You can also submit an amended application if you’ve made a mistake in the application, but the case has not progressed beyond the ‘acknowledgement of service’ stage.\r\n\r\n",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelAmendPetitionForRefusalSolPara-3",
+    "Label": "When you start an amended application, the old divorce case will be stopped and a new case will be created. You’ll still be able to access the old case by clicking on the ‘linked’ cases tab in the new case.\r\n\r\n",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelAmendPetitionForRefusalSolPara-4",
+    "Label": "The new case will be automatically populated with information from the old case. Please update the parts of the application that need to be different.\r\n\r\n",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelAmendPetitionForRefusalSolPara-5",
+    "Label": "The fee for an amended application is £95.\r\n\r\n",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "labelAmendPetitionForRefusalSolPara-6",
+    "Label": "Once the amended application has been submitted, it will be checked by the courts before being issued to the petitioner’s husband/wife so they can respond.\r\n\r\n",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
```

Comment:
Adding page content separated in paragraphs as in AC. Content can be found [here](https://docs.google.com/document/d/1U5E_DhAX1FdcIg5ukZRCcZ2XVPwTKBeYbLf4Lm9T3_k/edit?usp=sharing) and also in the Jira ticket. Definitions are added in the non-prod file